packages/manager/modules/octavia-load-balancer/src/delete/delete.routing.js

```diff
@@ -0,0 +1,39 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.delete', {
+    url: '/delete?loadBalancerId&loadBalancerName&loadBalancerRegion',
+    views: {
+      modal: {
+        component: 'octaviaLoadBalancerDelete',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      breadcrumb: () => null,
+      serviceName: /* @ngInject */ ($transition$) =>
```

Comment:
You can already retrieve projectId from parent resolve so no need of this line

Reply:
Done